server.ts

```diff
@@ -83,15 +83,20 @@ async function main() {
       } else if (CHAIN_EVENTS) {
         chains = CHAIN_EVENTS.split(',');
       }
-      const subscribers = await setupChainEventListeners(models, null, chains, SKIP_EVENT_CATCHUP);
+
+      await setupChainEventListeners(models, null, chains, SKIP_EVENT_CATCHUP);
+      // TODO: code below is deprecated since we no longer return subscribers
+
+      // const subscribers = await setupChainEventListeners(models, null, chains, SKIP_EVENT_CATCHUP);
+
       // construct storageFetchers needed for the identity cache
-      const fetchers = {};
-      for (const [ chain, subscriber ] of Object.entries(subscribers)) {
-        if (chainSupportedBy(chain, SubstrateTypes.EventChains)) {
-          fetchers[chain] = new SubstrateEvents.StorageFetcher(subscriber.api);
-        }
-      }
-      await identityFetchCache.start(models, fetchers);
+      // const fetchers = {};
+      // for (const [ chain, subscriber ] of Object.entries(subscribers)) {
+      //   if (chainSupportedBy(chain, SubstrateTypes.EventChains)) {
+      //     fetchers[chain] = new SubstrateEvents.StorageFetcher(subscriber.api);
+      //   }
+      // }
+      // await identityFetchCache.start(models, fetchers);
```

Comment:
We need to figure out how to deal with the loss of this identity fetch cache before we can get this PR in. It's used only in one place: `/updateProfile`, to fetch the Substrate identity when a user's address is provided. We will need to consider some options on how to replace this.

Reply:
With the recent push the cache is now in the database in the IdentityCaches table. Chain-events nodes have 2 options:

1. Set `HANDLE_IDENTITY` to "handle" to allow the chain-events nodes to update the database directly when they have fetched an identity from the cache.
2. Set `HANDLE_IDENTITY` to "publish" to make the chain-events nodes publish the identity data they fetch into an "identityQueue". The identity queue consumer is started alongside the regular chain-events consumer in `setupChainEventListeners.ts`.